en/JournalAbbreviations.md

```diff
@@ -31,23 +35,20 @@ Any entry in your personal journal list will override an entry with the same ful
 
 ### Your personal journal abbreviations list
 
-Your personal journal list is managed in the central part of the **Manage journal abbreviations** window. To start building your personal journal abbreviations list, choose *New file*, and enter a filename manually or using the *Browse* button. If you already have a file that you want to use as a starting point, use the *Browse* button on the line starting wiht *Existing file*. The table will update to show the contents of the list you have selected.
+Your personal journal list is managed on top of the **Manage journal abbreviations** window. To start building your personal journal abbreviations list, choose *Add new list*, and enter a filename. If you already have a file that you want to use as a starting point, use the *Open existing list* button. The table will update to show the contents of the list you have selected.
 
-The table and the tool buttons to the right allow you to add, remove and edit journal entries. For each entry you must provide the full journal name, and the ISO abbreviation (e.g. "Aquacultural Engineering" and "Aquacult. Eng."). To edit an entry, double-click its row in the table.
+The table and the tool buttons in the upper right allow you to add, remove and edit journal entries. For each entry you must provide the full journal name, and the default abbreviation (e.g. "Aquacultural Engineering" and "Aquacult. Eng."). The last field, which contains the shortest unique abbreviation, is optional. Therefore, you can actually safely omit it. To edit an entry, double-click its row in the table.
 
-Once you click *OK*, if you have selected a file, and the table contains at least one entry, the table contents will be stored to the selected file, and JabRef's list of journals will be updated.
+Once you click *Save changes*, if you have selected a file, and the table contains at least one entry, the table contents will be stored to the selected file, and JabRef's list of journals will be updated.
 
 ### External journal lists
 
-You can link to a number of external lists. These links can be set up in the lower part of the **Manage journal abbreviations** window. External lists are similar to the personal list - the only difference is that JabRef does not provide an interface for editing the external lists.
-
-External lists can be found at [JabRef's repository abbreviation lists](http://abbrv.jabref.org/).
+You can link to a number of external lists. These links can be set up on top of the **Manage journal abbreviations** window. External lists are similar to the personal list. The *Open existing list* button allows you to select an existing file on your computer.
 
-To add a new external list, if necessary, click the **+** button to add another slot to the interface. Then use either the *Browse* or *Download* button next to one of the slots in the lower part of the window:
+![External list](./images/JournalAbbreviations-ExternalList.png)
 
```

Comment:
Maybe add a hin that these are now in CSV format and how the CSV format is build (just a sample line maybe)

Reply:
It is done. Thanks!